api/models.py

```diff
@@ -1,3 +1,26 @@
 from django.db import models
+from datetime import date
 
-# Create your models here.
+
+class User(models.Model):
+    name = models.CharField('Nome', max_length=50)   
+    last_login = models.DateField(default=date.today)
```

Comment:
O DateField dá (year = 2018, month = 7, day = 12)
e o DateTimeField dá (year = 2018, month = 7, day = 12, hour = 7, minute = 9, second = 33)
Eu posso tá errada. Mas se for isso mesmo, seria melhor usar DateTimeField, vcs concordam?
Assim: last_login = models.DateTimeField('last_login',auto_now=True)